packages/common/src/plugins/requestParser/POSTMultipart.ts

```diff
@@ -12,23 +12,37 @@ export function isPOSTMultipartRequest(request: Request): boolean {
 export async function parsePOSTMultipartRequest(
   request: Request,
 ): Promise<GraphQLParams> {
-  const requestBody = await request.formData()
-  const operationsStr = requestBody.get('operations')?.toString() || '{}'
-  const operations = JSON.parse(operationsStr)
-  const mapStr = requestBody.get('map')?.toString() || '{}'
-  const map = JSON.parse(mapStr)
-  for (const fileIndex in map) {
-    const file = requestBody.get(fileIndex)
-    const keys = map[fileIndex]
-    for (const key of keys) {
-      dset(operations, key, file)
+  try {
+    const requestBody = await request.formData()
+    const operationsStr = requestBody.get('operations')?.toString() || '{}'
+    const operations = JSON.parse(operationsStr)
+    const mapStr = requestBody.get('map')?.toString() || '{}'
+    const map = JSON.parse(mapStr)
+    for (const fileIndex in map) {
+      const file = requestBody.get(fileIndex)
+      const keys = map[fileIndex]
+      for (const key of keys) {
+        dset(operations, key, file)
+      }
     }
-  }
 
-  return {
-    operationName: operations.operationName,
-    query: operations.query,
-    variables: operations.variables,
-    extensions: operations.extensions,
+    return {
+      operationName: operations.operationName,
+      query: operations.query,
+      variables: operations.variables,
+      extensions: operations.extensions,
+    }
+  } catch (e: any) {
```

Comment:
This try/catch block can be moved around  line 16 `const requestBody = await request.formData()` exclusively.

Reply:
Done